app/controllers/application_controller.rb

```diff
@@ -50,6 +51,10 @@ def require_subscription
     redirect_to root_path, notice: 'サブスクリプション登録が必要です。' unless current_user&.subscription?
   end
 
+  def require_scheduler_inheritation
+    head :internal_server_error if request.path_info.start_with?('/scheduler') && !is_a?(SchedulerController)
```

Comment:
```ruby
before_action :require_scheduler_inheritation, if: -> { request.path_info.start_with?('/scheduler') }
```

とした上で

```suggestion
    head :internal_server_error unless is_a?(SchedulerController)
```

とした方がコードの意図が明確になりそうな気がしました

Reply:
こちらで修正しました。
[5ba8640](https://github.com/fjordllc/bootcamp/pull/6970/commits/5ba8640de27f42c1be824df083ed96f7f7884050)